bibliography.bib

```diff
@@ -381,4 +381,19 @@ @inproceedings{hewitt-manning-2019-structural
+}
+
+@article{FRANCK2006173,
+title = {Agreement and movement: A syntactic analysis of attraction},
+journal = {Cognition},
+volume = {101},
+number = {1},
+pages = {173-216},
+year = {2006},
+issn = {0010-0277},
+doi = {https://doi.org/10.1016/j.cognition.2005.10.003},
+url = {https://www.sciencedirect.com/science/article/pii/S0010027705001861},
+author = {Julie Franck and Glenda Lassi and Ulrich H. Frauenfelder and Luigi Rizzi},
+keywords = {Sentence production, Subject–verb agreement, Attraction, Movement, Experimental psycholinguistics, Formal syntax},
+abstract = {This paper links experimental psycholinguistics and theoretical syntax in the study of subject–verb agreement. Three experiments of elicited spoken production making use of specific characteristics of Italian and French are presented. They manipulate and examine its impact on the occurrence of ‘attraction’ errors (i.e. incorrect agreement with a word that is not the subject of the sentence). Experiment 1 (in Italian) shows that subject modifiers do not trigger attraction errors in free inverted VS (Verb Subject) structures, although attraction was found in VS interrogatives in English (Vigliocco, G., & Nicol, J. (1998). Separating hierarchical relations and word order in language production. Is proximity concord syntactic or linear? Cognition, 13–29) In Experiment 2 (in French), we report stronger attraction with preverbal clitic object pronouns than with subject modifiers. Experiment 3 (in French) shows that displaced direct objects in the cleft construction trigger attraction effects, in spite of the fact that the object does not intervene between the subject and the verb in the surface word order (OSV). Moreover, attraction is stronger in structures with subject–verb inversion (...). These observations are shown to be naturally interpretable through the tools of formal syntax, as elaborated within the Principles and Parameters/Minimalist tradition. Three important constructs are discussed: (1) the hierarchical representation of the sentence during syntactic construction, and the role of intermediate positions by which words transit when they move; (2) the role of specific hierarchical (c-command) but also linear (precedence) relations; and (3) the possibility that agreement involves two functionally distinct components. A gradient of computational complexity in agreement is presented which relates empirical evidence to these theoretical constructs.}
```

Comment:
The abstract of Franck et al 2006 included in the bibtex here and quoted for its definition of "attraction errors" in the context of subject-verb agreement is reprinted from Cognition, 101, Julie Franck and Glenda Lassi and Ulrich H. Frauenfelder and Luigi Rizzi, "Agreement and movement: A syntactic analysis of attraction", pages 173 to 216.,
Copyright 2006, with explicit permission from Elsevier for publishing on a public website, purchased via RightsLink at copyright.com .